frontend/src/lib/components/TaxonomicFilter/taxonomicFilterLogic.tsx

```diff
@@ -195,6 +195,7 @@ export const taxonomicFilterLogic = kea<taxonomicFilterLogicType>([
                         getValue: (eventDefinition: Record<string, any>) =>
                             // Use the property's "name" when available, or "value" if a local option
                             'id' in eventDefinition ? eventDefinition.name : eventDefinition.value,
+                        excludedProperties: excludedProperties[TaxonomicFilterGroupType.Events],
```

Comment:
What do we need this for?

Reply:
it should be to exclude events from the search, though I don't think it actually works. I'll remove it